src/module/item/melee/sheet.ts

```diff
@@ -13,6 +13,29 @@ export class MeleeSheetPF2e extends ItemSheetPF2e<MeleePF2e> {
             attackEffects: createSheetOptions(this.getAttackEffectOptions(), item.system.attackEffects),
         };
     }
+
+    override activateListeners($html: JQuery<HTMLElement>): void {
+        super.activateListeners($html);
+
+        // Add Damage Roll
+        $html.find(".add-damage").on("click", (event) => {
+            event.preventDefault();
+            const newKey = randomID(20);
+            const newDamageRoll = { damage: "", damageType: "" };
```

Comment:
Maybe give this a default, like 1d4 bludgeoning

Reply:
I think that'll make it more work to enter data, cause now they'll have to delete it right?

Reply:
It'll be valid, at least :shrug: